### DISCUSSION TITLE
All users get notified when server is used as print server

### DISCUSSION THREAD

**Discussion by adityamj** [ltsp/ltsp#458] [Q&A]
Hi,

I have been successful in setting up printing with pointers from #158 and #221.
Clients that have directly attached usb printers use the p910nd method to expose port 9100 which is then configured on the server.
All printers are configured on the server, and shared.
Every thing works!

The minor (or major depending what is being printed) issue is that each user gets a notification of every print job on every configured printer on the server.

Every user is also able to see the entire queue. 

Is there a way to suppress this information? 

I am using Ubuntu 20.04 Mate edition on server with chrootless image.

Thank you!

**Comment by alkisg** [comment]
If you're using different user accounts for each client, then this doesn't sound like an LTSP issue but like a MATE issue. Try to contact the MATE developers about it.
